exercises/complex-numbers/.meta/generator/complex_numbers_case.rb

```diff
@@ -0,0 +1,36 @@
+require 'generator/exercise_case'
+
+class ComplexNumbersCase < Generator::ExerciseCase
+
+  def workload
+    indent_text(4, testcase)
```

Comment:
I think there is an indent function that works on arrays you could use and not need to manage newlines yourself in the testcase method.